department-of-veterans-affairs/vets-website
# Handle homeAcreageValue in transform

## Summary

Values from inactive fields (ie, ones where a 'depends' function has returned false) are cleared in the platform function transformForSubmit. This function relies on the presence of the fields in the schema.properties for the page. `homeAcreageValue` was incorrectly missing from the schema.properties, causing it to be left in the submitted data when it should have been removed.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76678

## Testing done

- Fill out the form at http://localhost:3001/pension/application/527EZ. In the financial information section, say that you own a home, it is over 2 acres, and it has a value of $20,000. 
- On the review page, edit the data to state the you do not own your own home. Don't change any other data. You will see the question about the home acreage value disappear.
- Submit the data, and view the form data sent (using the network tab). If on the `main` branch, you will see that `homeAcreageValue` was submitted even though it was removed from view on the review page. If you are on this branch, you will see that `homeAcreageValue` was removed from the submitted data. 

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
